yt/go/yt/integration/operations_test.go

```diff
@@ -131,6 +131,65 @@ func TestOperationWithStderr(t *testing.T) {
 	}
 }
 
+func TestSuspendOperation(t *testing.T) {
+	t.Parallel()
+
+	env := yttest.New(t)
+
+	ctx := ctxlog.WithFields(context.Background(), log.String("subtest_name", t.Name()))
+	ctx, cancel := context.WithTimeout(ctx, time.Minute)
+	defer cancel()
+
+	opSpec := map[string]any{
+		"tasks": map[string]any{
+			"main": map[string]any{
+				"job_count": 1,
+				"command":   "sleep 5m",
+			},
+		},
+	}
+	opID, err := env.YT.StartOperation(ctx, yt.OperationVanilla, opSpec, nil)
+	require.NoError(t, err)
+
+	// Wait for the job to start.
+	for idx := 0; idx < 10; idx++ {
```

Comment:
I suggest waiting here for the operation state to be `Running`. And then we could check that `require.Equal(t, false, opStatus.Suspended)`.